core/src/main/java/eu/ostrzyciel/jelly/core/internal/NameDecoderImpl.java

```diff
@@ -123,8 +123,11 @@ public TIri decode(RdfIri iri) {
             nameEntry = nameLookup[lastNameIdReference];
         } catch (ArrayIndexOutOfBoundsException e) {
             throw JellyExceptions.rdfProtoDeserializationError(
-                    "Encountered an invalid name table reference (out of bounds). " +
-                            "Name ID: " + nameId + ", Prefix ID: " + iri.prefixId()
+                "Encountered an invalid name table reference (out of bounds). " +
```

Comment:
btw format and multiline strings can be used to make it look a tad nicer :)
```java
var a = 1;
var b = 2;
var s = """
Yes i'm multiline.
Here's a param (%s)
And another one (%s)
""".formatted(a, b);
```
is a valid core for j17

Reply:
good thing intellij has a quickfix for that, otherwise I'd be too lazy to do it, thanks :D